Misc/Copy-ItemToBcContainer.ps1

```diff
@@ -0,0 +1,66 @@
+﻿<# 
+ .Synopsis
+  Copy an item (file or folder) from the host system to a container.
+ .Description
+  Copy an item (file or folder) from the host system to a container.
+ .Parameter containerName
+  Name of the container to which you want to copy the item.
+ .Parameter localPath
+  Path to the item on the host system which has to be copied.
+ .Parameter containerPath
+  Path in the container where the item will be placed.
+  If the source item is a folder, 'containerPath' will be created (or treated) as a folder and all items inside the source folder will be copied to it.
+ .Example
+  Copy-ItemToBcContainer -containerName test2 -localPath "c:\temp\myfile.txt" -containerPath "c:\run\my\myfile.txt"
+#>
+function Copy-ItemToBcContainer {
+    Param (
+        [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
+        [Parameter(Mandatory = $true)]
+        [string] $localPath,
+        [Parameter(Mandatory = $false)]
+        [string] $containerPath = $localPath
+    )
+
+    $telemetryScope = InitTelemetryScope -name $MyInvocation.InvocationName -parameterValues $PSBoundParameters -includeParameters @()
+    try {
+
+        if (!(Test-BcContainer -containerName $containerName)) {
+            throw "Container $containerName does not exist"
+        }
+        Write-Host "Copy '$localPath' from host to '$containerPath' on container $containerName"
```

Comment:
also test here whether containerPath exists - if it does, fail early

Reply:
Same as in `Copy-ItemFromBcContainer` more or less. But I'd rather not fail at all here. I imagine a scenario where I stage multiple files from the host into the container with multiple calls. Failing would force me to stage on the host first and then copy to the container with one single call.

Reply:
got it and agree